schema/boundingVolume.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "Tile",
+    "type" : "object",
+    "description" : "A bounding volume that encloses a tile or its contents.  At least one property is required.  If more than one property is defined, the runtime can determine which to use.",
+    "properties" : {
+        "box" : {
+            "type" : "array",
+            "description" : "An array of 12 numbers that define a oriented bounding box in WGS84 coordinates.  The first three elements define the x, y, and z values for the center of the box.  The next three elements (with indices 3, 4, and 5) define the x axis direction and length.  The next three elements (indices 6, 7, and 8) define the y axis direction and length.  The last three elements (indices 9, 10, and 11) define the z axis direction and length.",
```

Comment:
Not quite sure I understand this. The first three numbers is the center vector, and the following groups of three numbers are vectors that specify lengths and directions of the box's sides, is that right? Does `axis` here refer to the box's own axes?